templates/zerver/markdown_help.html

```diff
@@ -1,105 +1,105 @@
-          <td>Some inline <code>code</code></td>
-        </tr>
-        <tr>
-          <td class="preserve_spaces">```
+                <tbody>
+                    <tr>
+                        <td>*italic*</td>
+                        <td><i>italic</i></td>
+                    </tr>
+                    <tr>
+                        <td>**bold**</td>
+                        <td><b>bold</b></td>
+                    </tr>
+                    <tr>
+                        <td>~~strikethrough~~</td>
+                        <td><del>strikethrough</del></td>
+                    </tr>
+                    <tr>
+                        <td>[Zulip website](https://zulip.org)</td>
+                        <td><a href="https://zulip.org" target="_blank">Zulip website</a></td>
+                    </tr>
+                    <tr>
+                        <td>* Archimedes<br/>
+                            * Bohr<br/>
+                            * Curie
+                        </td>
+                        <td>
+                            <ul>
+                                <li>Archimedes</li>
+                                <li>Bohr</li>
+                                <li>Curie</li>
+                            </ul>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>:heart: (and <a href="http://www.emoji-cheat-sheet.com/" target="_blank">many others</a>, from the <a href="https://code.google.com/p/noto/" license="/static/generated/emoji/images/emoji/NOTICE" target="_blank">Noto Project</a>)</td>
+                        <td><img alt=":heart:" class="emoji" src="/static/generated/emoji/images/emoji/heart.png" title=":heart:" /></td>
+                    </tr>
+                    <tr>
+                        <td>@**Joe Smith**<br/>
+                          (autocompletes from @joe)</td>
+                        <td><span class="user-mention">@Joe Smith</span> (notifies Joe Smith)</td>
+                    </tr>
+                    <tr>
+                        <td>@**all**</td>
+                        <td><span class="user-mention">@all</span> (notifies all recipients)</td>
+                    </tr>
+                    <tr>
+                        <td>#**streamName**</td>
+                        <td><a>#streamName</a> (links to a stream)</td>
+                    </tr>
+                    <tr>
+                        <td>Some inline `code`</td>
+                        <td>Some inline <code>code</code></td>
+                    </tr>
+                    <tr>
+                      <td class="preserve_spaces">```
 def zulip():
     print "{{product_name}}"
 ```</td>
-          <td><pre>def zulip():
+                      <td><pre>def zulip():
     print "{{product_name}}"</pre></td>
-        </tr>
-        <tr>
-          <td class="preserve_spaces">``` .py
+                    </tr>
+                    <tr>
+                      <td class="preserve_spaces">``` .py
```

Comment:
That is a manual error. markdown_help actually needs manual editing after running tool.
Will fix it.